nebula_ros/include/nebula_ros/continental/continental_ars548_hw_interface_wrapper.hpp

```diff
@@ -26,6 +26,7 @@
 #include <continental_srvs/srv/continental_ars548_set_vehicle_parameters.hpp>
 #include <geometry_msgs/msg/accel_with_covariance_stamped.hpp>
 #include <geometry_msgs/msg/twist_with_covariance_stamped.hpp>
+#include <std_msgs/msg/float32.hpp>
```

Comment:
Did ARS548 changes leak from somewhere?

Reply:
For some reason a bunch of PRs did not compile due to this import missing... maybe some other included header had been importing it and after some change, it didn't anymore and so this file threw an error

Reply:
For some reason a bunch of PRs did not compile due to this import missing... maybe some other included header had been importing it and after some change, it didn't anymore and so this file threw an error

Reply:
For some reason a bunch of PRs did not compile due to this import missing... maybe some other included header had been importing it and after some change, it didn't anymore and so this file did not compile anymore

Reply:
Just double checked. Both develop and this branch without this change compile